go.mod

```diff
@@ -29,7 +30,7 @@ require (
 	github.com/integr8ly/application-monitoring-operator v1.6.0
 	github.com/integr8ly/cloud-resource-operator v0.30.0
 	github.com/integr8ly/grafana-operator v2.0.0+incompatible
-	github.com/integr8ly/grafana-operator/v3 v3.10.2
+	github.com/integr8ly/grafana-operator/v3 v3.10.3
```

Comment:
This is possibly causing some issue. The grafana operator needs to be v3.10.4 to remove the v1beta1